tenable/io/v3/base/endpoints/explore.py

```diff
@@ -77,28 +119,17 @@ def search(
                 If ``return_json`` was set to ``True``, then a response
                 object is instead returned instead of an iterable.
 
-        Examples:
-
-            >>>
         '''
-        kwargs['fields'] = fields
-        kwargs['sort'] = sort
-        kwargs['filter'] = filter
-        kwargs['limit'] = limit
-        kwargs['next'] = next
-        if not schema_cls:
-            schema_cls = SearchSchema
-        if not iterator_cls:
-            # todo - commenting this temporarily
-            # iterator_cls = ExploreSearchIterator
-            pass
```

Comment:
This should likely be removed as you are specifying the classes as default above.

Reply:
these are removed from the code base.